### PR TITLE
Implement cobra for command line

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
@gabemontero ptal

File: main.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package main
```
Comment (gabemontero) [inline main.go:1]: Yeah i'm not fan of main.go in the top of the repo

move it to the cmd folder and employ filenames similar to https://github.com/openshift/builder/tree/master/cmd

Comment (coreydaley) [reply main.go]: updated in current push, ptal

**Comment by coreydaley** [comment]
/hold
Should probably wait for https://github.com/openshift/csi-driver-projected-resource/pull/15 to merge and then I can rebase this one if needed.

File: Makefile [hunk 2]
```diff
@@ -31,7 +31,7 @@ verify: ## Run verifications. Example: make verify
 .PHONY: verify
 
 build: ## Build the executable. Example: make build
-	go build -a -race -ldflags $(LDFLAGS) -o _output/hostpathplugin ./cmd/hostpathplugin
+	go build -a -mod=vendor -race -ldflags $(LDFLAGS) -o csi-driver-projected-resource ./cmd
```
Comment (gabemontero) [inline Makefile:34]: if you put it here, then we have to deal with our .gitignore and the new binary

whereas the _output dir is a convention we already have in place for openshift

**Review by gabemontero — CHANGES_REQUESTED** [review]
one minor item on binary location

File: Makefile (hunk 2 above)
Comment (coreydaley) [reply Makefile]: update pushed.

**Pull request merged**